README.md

```diff
@@ -50,7 +50,7 @@ github "YUNEEC/Yuneec-SDK-iOS" "branch-name"
 
 ## Docs
 
-Autogenerated API docs can be found in the [docs](docs/) directory.
+Autogenerated API docs can be found in the [docs]() directory.
```

Comment:
Does this link work?

Reply:
Hmm, so the issue is that the docs generated includes the ReadMe as the main content of the Index.html. The link I changed works to open the docs folder on the users local machine. However, the previous link allows users to click docs from the ReadMe as displayed on github. I think the best option would actually be a link to the docs page from the developer portal, kind of a self referencing link. This would direct all clicks on that link to http://developer.yuneec.com/sites/default/files/ios-sdk/index.html. Thoughts?

Reply:
What about something like:
```
Autogenerated API docs can be found locally in the [docs](docs) directory or on the [developer portal](https://developer.yuneec.com/sites/default/files/ios-sdk/index.html).
```

Reply:
That's perfect